packages/loader/driver-utils/src/bufferToString.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IsoBuffer } from "@fluidframework/common-utils";
+
+/**
+ * Convert binary blob to string format
+ *
+ * @param blob - the binary blob
+ * @returns the blob in string format
+ */
+export const bufferToString = (blob: ArrayBufferLike): string => IsoBuffer.from(blob).toString("utf8");
```

Comment:
Any reason this API is here, and not in common-utils next to toBuffer()?
It's expected that for staging you need in both places (common-utils and driver-utils), but why (in both places) it's not in the same file as toBuffer?